python/ray/autoscaler/v2/tests/test_e2e.py

```diff
@@ -200,6 +207,111 @@ def verify():
         cluster.shutdown()
 
 
+def test_object_store_memory_idle_node(shutdown_only):
+
+    ray.init()
+
+    obj = ray.put("hello")
+
+    def verify():
+        state = get_cluster_status()
+        for node in state.healthy_nodes:
+            assert node.node_status == "RUNNING"
+            assert node.used_resources()["object_store_memory"] > 0
+            return True
+
+    wait_for_condition(verify)
+
+    del obj
+
+    import time
+
+    time.sleep(1)
```

Comment:
why do we need sleep here?

Reply:
to assert on the idle time has an impact.